### PR TITLE
Travis 2

### PR DESCRIPTION
**Pull request by arstrube**

### CODE REVIEW

File: lib/travis_github_deployer/travis_github_deployer.rb [hunk 1]
```diff
@@ -66,17 +70,17 @@ def load_configuration
...
-      
-      if source_files.empty?
-        raise StandardError.new("File: '#{source_file_from_configuration}' found in the configuration didn't exist. Deploy failed.") 
+  def prepare_files_to_deploy file_sequence
+    file_sequence.each { |file|
+      sources = Dir.glob(file["source"])
+      if sources.empty?
+        raise StandardError.new("File: '#{file["source"]}' found in the configuration didn't exist. Deploy failed.") 
+      end
+      if file["purge"] == "yes"
```
Comment (basvodde) [inline lib/travis_github_deployer/travis_github_deployer.rb]: Perhaps best to use a nocasecmp here?

File: lib/travis_github_deployer/travis_github_deployer.rb [hunk 2]
```diff
@@ -66,17 +70,17 @@ def load_configuration
...
+         files_to_purge.push(file["source"])
```
Comment (basvodde) [inline lib/travis_github_deployer/travis_github_deployer.rb]: More commonly this is written as file_to_purge << file["source"]

File: spec/enable_rspec_should_syntax.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+RSpec.configure do |config|
```
Comment (basvodde) [inline spec/enable_rspec_should_syntax.rb]: I don't think we want this one here :) Either live with the warnings or change the spec format to the rspec3. You can do it automatically with http://yujinakayama.me/transpec/
(If you do, please make that a separate pull request)

**Comment by basvodde** [comment]
Some ruby-ish recommendations. Rest looks ok as far as I can see.

File: lib/travis_github_deployer/travis_github_deployer.rb (hunk 1 above)
Comment (arstrube) [inline lib/travis_github_deployer/travis_github_deployer.rb]: of course.

File: spec/enable_rspec_should_syntax.rb (hunk 3 above)
Comment (arstrube) [inline spec/enable_rspec_should_syntax.rb]: I hate the warnings. and I love the 'should' syntax :-(

File: lib/travis_github_deployer/travis_github_deployer.rb (hunk 2 above)
Comment (arstrube) [inline lib/travis_github_deployer/travis_github_deployer.rb]: ah, kewl

**Comment by arstrube** [comment]
Thanks for the recommendations. Much appreciated.
